classes/opponentPhaseTeleport.lua

```diff
@@ -23,7 +23,7 @@ function opponentPhaseTeleport.new()
 	self.timeToTele = 0
 
 	self.xspeed = 0
-	self.yspeed = 0
+	self.yspeed = MAX_SPEED / 2
```

Comment:
Similar reason for the above ones. I added this so that the teleport phase starts off moving down, so that its angle doesn't automatically switch to facing the player the frame after ending the first teleport